utils/build.ps1

```diff
@@ -3434,19 +3439,21 @@ if (-not $SkipBuild) {
   Invoke-BuildStep Build-LLBuild $HostPlatform
   Invoke-BuildStep Build-ArgumentParser $HostPlatform
   Invoke-BuildStep Build-Driver $HostPlatform
-  Invoke-BuildStep Build-Crypto $HostPlatform
-  Invoke-BuildStep Build-Collections $HostPlatform
-  Invoke-BuildStep Build-ASN1 $HostPlatform
-  Invoke-BuildStep Build-Certificates $HostPlatform
-  Invoke-BuildStep Build-System $HostPlatform
-  Invoke-BuildStep Build-Build $HostPlatform
-  Invoke-BuildStep Build-PackageManager $HostPlatform
-  Invoke-BuildStep Build-Markdown $HostPlatform
-  Invoke-BuildStep Build-Format $HostPlatform
-  Invoke-BuildStep Build-LMDB $HostPlatform
-  Invoke-BuildStep Build-IndexStoreDB $HostPlatform
-  Invoke-BuildStep Build-SourceKitLSP $HostPlatform
-  Invoke-BuildStep Build-Inspect $HostPlatform
+  if (-not(Compare-Object $Test @("lldb"))) {
```

Comment:
I'm not sure that this is a good idea to do. I think that it would be better to fork the script and create a new divergent copy. We do not want developers to have different ways to build based upon what they are testing. Testing is an after thought in this script, this is meant to be for building a toolchain distribution.

Reply:
Also, note that this doesn't work for me - I can be testing locally and want a complete toolchain build.

Reply:
>  I think that it would be better to fork the script and create a new divergent copy.

I agree, testing lldb only seems too specific for this script.

My concern with forking the script is having multiple copies of the same functions, like `Build-Compilers`. What do you think of moving some of `build.ps1`'s function in a common file that can be imported by either `build.ps1` or `test-lldb.ps1`?